share/smack/include/pthread.h

```diff
@@ -0,0 +1,71 @@
+//
```

Comment:
I'll go through the examples today and tomorrow, and make comments for each of the files regarding suitability, etc.

Let me know when you want to sit down.